src/Console/Commands/AuthMigration.php

```diff
@@ -25,8 +25,8 @@ public function __construct(Composer $composer)
     public function handle()
     {
         $from = __DIR__.'/stubs/auth/statamic_auth_tables.php.stub';
-        $file = date('Y_m_d_His', time()).'_statamic_auth_tables';
-        $to = database_path("migrations/{$file}.php");
+        $file = date('Y_m_d_His', $this->option('test') ? 1648622397 : time()).'_statamic_auth_tables';
```

Comment:
Why not import `Illuminate\Support\Carbon` and use `Carbon::now()->format('Y_m_d_His')`?
That way you won't need to add a `--test` option.